tools/mage/build_namespace.go

```diff
@@ -120,7 +121,8 @@ func (b Build) Lambda() {
 		}
 	})
 
-	logger.Infof("build:lambda: compiling %d Go Lambda functions (internal/.../main)", len(packages))
+	logger.Infof("build:lambda: compiling %d Go Lambda functions using %s (internal/.../main)",
```

Comment:
Nit: the paths relate to the compilation, not the Go version:

```suggestion
	logger.Infof("build:lambda: compiling %d Go Lambda functions (internal/.../main) using %s",
```

Reply:
done